### ISSUE TITLE
Drop the TFM column from output

### ISSUE THREAD

**Issue by AArnott** [terrajobst/apisof.net#29]
Now with #25 completed, the `net6.0-windows` column is purely wasted space. It only says "Unavailable".

**Comment by terrajobst** [comment]
That is problematic for two reasons:

1. **Multiple frameworks**. If you specify more than one framework you'll want to know where it's not available. Why multiple framework? Because many parties porting need to do it in stages and typically ask about .NET Core and .NET Standard 2.0 together:

   ```bash
   $ apisofdotnet check-apis D:\myapp -t net6.0 -t netstandard2.0 -o result.csv
   ```

   I could special a single framework, but I'm not sure this would make it better or worse due to the inconsistency.

2. **Packages**. I just fixed a bug in 21af0c93e421fd1230460046fc751891295010e2 where I excluded rows where the API is available but requires a package. In the initial planning this might or might not be a big deal, but I've seen people being confused when we didn't tell them that they need a package.
